TwilioVerifySDKTests/TwilioVerify/Sources/API/FactorAPIClientTests.swift

```diff
@@ -457,8 +466,10 @@ extension FactorAPIClientTests {
       accountSid: Constants.accountSid,
       serviceSid: Constants.serviceSid,
       identity: Constants.identity,
+      allowIphoneMigration: false,
       createdAt: Date(),
-      config: Config(credentialSid: Constants.credentialSid))
+      config: Config(credentialSid: Constants.credentialSid)
+    )
     static let failureResponse = FailureResponse(
       statusCode: 401,
       errorData: "error".data(using: .utf8)!,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/a8a0ffb2344905f9a27850246dcbbe000db39834/TwilioVerifySDKTests/TwilioVerify/Sources/API/FactorAPIClientTests.swift#L475"></span>Prefer non-optional <code>Data(_:)</code> initializer when converting <code>String</code> to <code>Data</code><br />
<code>non_optional_string_data_conversion</code> <code>FactorAPIClientTests.swift:475</code></td>
    </tr>
  </tbody>
</table>